cartoframes/utils/metrics.py

```diff
@@ -79,11 +80,17 @@ def build_metrics_data(event_name):
         'runtime_env': get_runtime_env()
     }
 
+    if not isinstance(extra_metrics_data, dict):
```

Comment:
We need @cmongut @ilbambino for the definition of the metrics structure. Maybe they require "user_id" instead of a field called "extra", so I think we could merge both dictionaries (metrics_data + extra_metrics_data)

Reply:
Yes, it isn't defined. But the extra part is almost an error. The just uploaded commit it only adds the `extra` key if the `extra_metrics_data` is not a `dict`.

Reply:
I wouldn't add the `extra` key until it's required from `Product`, so I think we can remove the extra_metrics_data as  != dict and return just {}